src/Elmish.WPF/Binding.fs

```diff
@@ -94,20 +96,32 @@ module Binding =
     |> mapData (alterMsgStream alteration)
 
 
+  /// <summary>
+  ///   Strongly-typed bindings that update the view from the model.
+  ///   Can be used in binding lists if boxed using <see cref="boxT" />.
+  /// </summary>
   module OneWayT =
 
     /// Elemental instance of a one-way binding.
     let id<'a, 'msg> : string -> Binding<'a, 'msg, 'a> =
       OneWay.id
       |> createBindingT
 
+  /// <summary>
+  ///   Strongly-typed bindings that update the model from the view.
+  ///   Can be used in binding lists if boxed using <see cref="boxT" />.
```

Comment:
Remove this line as well. The `OneWayToSource` module literally does exactly this.